tests/unittests/test_datasource/test_azure.py

```diff
@@ -1285,9 +1344,10 @@ def test_fallback_network_config_blacklist(self, mock_fallback, mock_dd,
         self.assertTrue(ret)
 
         netconfig = dsrc.network_config
-        expected_config = fallback_config
-        expected_config['config'].append(blacklist_config)
```

Comment:
@blackboxsw Is `test_fallback_network_config_blacklist` still a necessary UT? I discovered that this UT previously never failed because `expected_config = fallback_config` was a shallow copy rather than a deep copy.
Additionally, the UT `test_imds_network_config` already tests that `generate_fallback_config` is called with blacklist drivers in the arguments.
Additionally, the UT `test_device_driver_blacklist` in `tests/unittests/test_net.py` already tests the functionality of the device driver blacklist.
Since this UT (`test_fallback_network_config_blacklist`) does not accomplish much, I am of the opinion that it should be removed. Thoughts?

Reply:
@johnsonshi Ugh, you are right on the shallow copy and I agree with you that we can drop `test_fallback_network_config_blacklist`  specifically because generate_fallback_config is actually tested in test_net.py as you mentioned. No need to integration test it from this side.